.github/workflows/eslint.yml

```diff
@@ -0,0 +1,11 @@
+name: ESLint
+on: [push, pull_request]
```

Comment:
Can you run it only on `pull_request`? Otherwise we will see two GitHub actions running for ESlint on each PR.

Reply:
@Ron-Lavi updated the changes: https://github.com/girijaasoni/newsletterApplicaton/pull/18/commits/75fae33b684986d689a7f92c00a35e90c5485210